rustls/src/server/server_conn.rs

```diff
@@ -207,7 +207,10 @@ impl<'a> ClientHello<'a> {
 /// * [`ServerConfig::send_tls13_tickets`]: 4 tickets are sent.
 ///
 /// [`RootCertStore`]: crate::RootCertStore
-pub struct ServerConfig<C: CryptoProvider> {
+pub struct ServerConfig {
+    /// Source of randomness and other crypto.
```

Comment:
Nit: I'd keep it at the bottom, seems pretty boring?

Reply:
going to move this to third, after the configured ciphersuites and kx-groups. it seems logical that it belongs next to these in the docs?

Reply:
Sure, that seems alright, too.